tests/test_bs_helper.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import sys
+import os
+
+from unittest.mock import patch, PropertyMock
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                             '..')))
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                             '..', 'smarsy')))
+# excluding following line for linter as it complains that
+# from import is supposed to be at the top of the file
+
+from smarsy.bs_helper import BSHelper # noqa
+
+
+class TestBSHelperInstance(unittest.TestCase):
+    def setUp(self):
```

Comment:
setup method for single test seems a bit like redundancy

Reply:
Done